### PR TITLE
implement LoggingArgFuncNotFullyAppliedAnalyzer

### PR DESCRIPTION
**Pull request by dawedawe**

### CODE REVIEW

**Review by nojaf — CHANGES_REQUESTED** [review]
Good progress!
I wouldn't mix the analyzers, I think you can work around it.

File: src/FSharp.Analyzers/LoggingArgFuncNotFullyAppliedAnalyzer.fs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+                        let argsContainPartiallyAppliedFunc =
+                            args
+                            |> List.exists (
+                                function
+                                | NewArray (_type, exprs) ->
+                                    exprs
+                                    |> List.exists (
+                                        function
+                                        | Coerce (_type, expr) ->
```
[the remark this answers is not in the document]
Comment (dawedawe) [reply src/FSharp.Analyzers/LoggingArgFuncNotFullyAppliedAnalyzer.fs]: I'm pleasantly surprised, this really seems to be enough :]

**Review by nojaf — APPROVED** [review]
Looks good. Shall we ship a new version with these?

**Comment by dawedawe** [comment]
> Looks good. Shall we ship a new version with these?

Yes, I think two new analyzers are enough for a new release. I added the changelog entries but need a new approval.

**Pull request merged**